README.md

```diff
@@ -1,16 +1,17 @@
-<p align="center">
+# Tiktok Ads Source dbt Package ([Docs](https://fivetran.github.io/dbt_tiktok_ads_source/))
+
+<p align="left">
     <a alt="License"
-        href="https://github.com/fivetran/dbt_ad_reporting/blob/main/LICENSE">
+        href="https://github.com/fivetran/dbt_tiktok_ads_source/blob/main/LICENSE">
         <img src="https://img.shields.io/badge/License-Apache%202.0-blue.svg" /></a>
     <a alt="dbt-core">
-        <img src="https://img.shields.io/badge/dbt_Core™_version->=1.6.0_<2.0.0-orange.svg" /></a>
+        <img src="https://img.shields.io/badge/dbt_Core™_version->=1.3.0_<2.0.0-orange.svg" /></a>
```

Comment:
The TikTok Ads example had a wider dbt Core™ version requirement; whereas, Ad Reporting is a bit more strict. Can you change this back to `1.6.0` for this README edit.